packages/bot-web-ui/src/components/dashboard/quick-strategy/quick-strategy-components/inputs.tsx

```diff
@@ -22,8 +22,9 @@ const Inputs = ({
     handleChange,
     onChangeInputValue,
     setCurrentFocus,
-}: TInputs) =>
-    is_input_field ? (
+}: TInputs) => {
+    const is_mobile = isMobile();
+    return is_input_field ? (
         <div
             className={classNames('quick-strategy__form-row', {
                 'quick-strategy__form-row--multiple': !is_mobile,
```

Comment:
```suggestion
                'quick-strategy__form-row--multiple': !is_mobile(),
```

Reply:
@prince-deriv I think it is better to call the function `isMobile()` only one time in the component. Here we need to use this variable a few times.  It is not good practice to do several calls without reason if we able to get boolean one time and use it throughout the component, where needed.  Where we need to get variable once in the component (is mobile version or not) I made a direct call.
Don't you think so?

Reply:
I completely agree...
If we have multiple occurrences of `is_mobile` then it is better to call the method only once at the top and reuse the variable.